Brokerages/Tradier/TradierBrokerage.DataQueueHandler.cs

```diff
@@ -43,9 +43,10 @@ public partial class TradierBrokerage
         private bool _disconnect;
         private volatile bool _refresh = true;
         private Timer _refreshDelay = new Timer();
-        private readonly ConcurrentDictionary<Symbol, string> _subscriptions = new ConcurrentDictionary<Symbol, string>();
         private Stream _tradierStream;
 
+        public IEnumerable<Symbol> Subscriptions => _subscriptionManager.GetSubscribedSymbols();
```

Comment:
Not sure we need this public property?

Reply:
It's used for new update ticker list and recreate session

Reply:
> It's used for new update ticker list and recreate session

But that's all within the same class (even if partial), so it can be made private?